packages/react/src/side-nav/SideNavLink.tsx

```diff
@@ -70,7 +78,26 @@ export function SideNavLink({
 				}}
 			>
 				<SideNavLinkIndicator depth={depth} />
-				<span>{label}</span>
+
+				<span css={{ flex: 1 }}>{label}</span>
```

Comment:
Personal opinion (and coming from an atomic class name world), but I don't like shorthand properties because I don't know what it's actually doing, or indeed, if you need all the things the shorthand sets

Reply:
- [ ] `flex-grow: 1`